Apromore-Core-Components/Apromore-Manager/pom.xml

```diff
@@ -91,8 +91,10 @@
                             org.apromore.service.helper,
                             org.apromore.service.model,
                             org.apromore.service.search,
-                            org.apromore.util
-                        </Export-Package>                        
+                            org.apromore.util,
+                            org.apromore.util.datetime
+                        </Export-Package>
+                        <JPA-PersistenceUnits>Apromore</JPA-PersistenceUnits>
                     </instructions>
                 </configuration>
             </plugin>
```

Comment:
Agree this is a good refactoring to do. Can we put it in a separate module called apromore-commons as discussed before? The common miscellaneous utilities like date, time, stats, math, string, files, etc. are good candidates to be in a "commons" package (like apache-commons), this is the first and right time to do that. Apromore-Manager is too big, will have to be refactored and thus shouldn't be added more.

Reply:
The JPA-PersistenceUnits manifest entry at line Apromore-Manager/pom/xml:97 was removed as part of Nolan's recent factoring-out of the persistence into Apromore-Database.  I think its return is unintended.